src/feature_transform_implementations.rs

```diff
@@ -568,7 +568,7 @@ impl TransformerCombine {
             },
         };
         let mut executor_from_namespaces: [ExecutorFromNamespace; 4] =
-            [c.clone(), c.clone(), c.clone(), c.clone()];
+            [c.clone(), c.clone(), c.clone(), c];
```

Comment:
@bbenshalom seems static analysis picked up a bound of 3 here, however, r.e. added the clone should we need this sometime in the future (works perfectly fine with it though)